14_array_extensions/array_extensions.rb

```diff
@@ -0,0 +1,15 @@
+class Array
+  def sum
+    s = 0
+    empty? ? s = 0 : each { |item| s += item }
```

Comment:
Try to use `each_with_object(0)` instead of `each` and remove that `s` variable. Also avoid using single-letter variables. `sum` is better.